resources/aws/subnet.go

```diff
@@ -20,39 +19,26 @@ type Subnet struct {
 	AWSEntity
 }
 
-func (s Subnet) findExisting() (*ec2.Subnet, error) {
-	subnets, err := s.Clients.EC2.DescribeSubnets(&ec2.DescribeSubnetsInput{
+func (s *Subnet) list() ([]*ec2.Subnet, error) {
```

Comment:
This was by mistake, however I think it's good practice to make all receivers consistent. Otherwise an instance will fulfil different interfaces depending on passing it by value or as a pointer. WDYT?